JitsiConference.js

```diff
@@ -80,6 +84,7 @@ function JitsiConference(options) {
     this.connectionIsInterrupted = false;
 
 }
+JitsiConference.prototype.constructor = JitsiConference;
```

Comment:
I see why this is needed, but given the magnitude of this change already, how about making JitsiConference an ES6 class as part of the process?